diff --git a/dselect/po/Makefile.in.in b/dselect/po/Makefile.in.in
index 83d8838..d9b4dae 100644
--- a/dselect/po/Makefile.in.in
+++ b/dselect/po/Makefile.in.in
@@ -9,7 +9,7 @@
 # General Public License and is *not* in the public domain.
 #
 # Origin: gettext-0.18
-GETTEXT_MACRO_VERSION = 0.18
+GETTEXT_MACRO_VERSION = 0.19
 
 PACKAGE = @PACKAGE@
 VERSION = @VERSION@
diff --git a/scripts/po/Makefile.in.in b/scripts/po/Makefile.in.in
index 83d8838..d9b4dae 100644
--- a/scripts/po/Makefile.in.in
+++ b/scripts/po/Makefile.in.in
@@ -9,7 +9,7 @@
 # General Public License and is *not* in the public domain.
 #
 # Origin: gettext-0.18
-GETTEXT_MACRO_VERSION = 0.18
+GETTEXT_MACRO_VERSION = 0.19
 
 PACKAGE = @PACKAGE@
 VERSION = @VERSION@
